### PR TITLE
Milestone 1: Components

### PR DESCRIPTION
**Pull request by Kingjosh007**
In this milestone, I had to initialize the project with components. Here is an overview of what I did:

- I created a React project using create-react-app;
- I installed the necessary packages, including `react-router-dom` and `react-redux` which will be useful later;
- I followed the instruction given [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps.md);
> I have 5 reusable components (building blocks) in total: Books, Book, BookForm, Categories, and Navbar; In the Navbar, I implemented the routing;
- I wrote a professional README file.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,14 +1,23 @@
-/* eslint-disable react/prefer-stateless-function */
-import React, { Component } from 'react';
-import './index.css';
-import Calculator from './components/Calculator';
+import './App.css';
```
Comment (Davidosky007) [inline src/App.js:1]: * I get an error when i run `npm start` command kindly fix this error HINT 💭  the file (App.css) you are trying to import is missing to fix the error you have to create the file or remove the import statement 👍 
image

File: src/components/Books.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import React from 'react';
+import Book from './Book';
+import BookForm from './BookForm';
+
+const Books = (props) => {
+  const { books } = props;
+  return (
+    <>
+      <ul className="listContainer">
+        {books.map((book) => (
```
Comment (Davidosky007) [inline src/components/Books.js:11]: * Kindly fix this error i get also after temporary fixing the first error i get this too 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @Kingjosh007 👋, 

Good job so far! 💪 
* Well written PR ✔️ 
* No linter errors ✔️ 
* Descriptive commit messages ✔️ 
* Proper use of git-flow ✔️ 
## There are some issues that you still need to work on to go to the next project but you are almost there!
almost-gif


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kingjosh007** [comment]
Thanks for the feedback. I worked on the changes. Now it works perfectly.

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @Kingjosh007 , 

Great Work!
Project works without any error :clap:
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**